frontend/app/views/spree/shared/_filters.html.erb

```diff
@@ -11,17 +11,14 @@
           <% labels.each do |nm,val| %>
             <% label = "#{filter[:name]}_#{nm}".gsub(/\s+/,'_') %>
             <li class="nowrap">
-              <input type="checkbox"
-                     id="<%= label %>"
-                     name="search[<%= filter[:scope].to_s %>][]"
-                     value="<%= val %>"
-                     <%= params[:search] && params[:search][filter[:scope]] && params[:search][filter[:scope]].include?(val.to_s) ? "checked" : "" %> />
+            <%= check_box_tag "search[#{filter[:scope].to_s}][]", val,
```

Comment:
I think the best way to indent this checkbox params is using this [style guide](https://github.com/thoughtbot/guides/blob/master/style/erb/sample.html.erb):

```erb
<%= check_box_tag(
  param_1,
  param_2,
  etc
) %>
```